.gitignore

```diff
@@ -4,7 +4,7 @@ build
 
 # Android Studio stuff
 .idea
-*.properties
+local.properties
```

Comment:
I have issues with that one. Since clientId and clientSecret origin from gradle.properties, this change means I'd either have to check those in (bad) or keep a permanent local diff (bad). Same applies to signing.properties.

We'd have to have a way to store client ID/client secret in some other properties file and then check in just gradle.properties, but I'm not sure about the best way towards that at the moment ... I'm not a Gradle expert :-/

Reply:
I missed that one...tell me if you're happy with the new commit.

Reply:
That one looks fine, thanks!